extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/BuildTimeEnabledProcessor.java

```diff
@@ -72,7 +72,7 @@ void unlessBuildProfile(CombinedIndexBuildItem index, BuildProducer<BuildTimeCon
     }
 
     @BuildStep
-    void ifBuildProperty(BeanArchiveIndexBuildItem index, BuildProducer<BuildTimeConditionBuildItem> producer) {
```

Comment:
This was bothering me so I looked into it more. If we do this, existing code could break, so we need to find another way